examples/cinematography_domain/client.py

```diff
@@ -17,8 +17,8 @@ class CineClient(DomainClient):
     `movie`) are create in the base class, and here we have only added type annotations so that IntelliSense can work.
     """
 
-    person: DomainModelAPI[Person]
     movie: DomainModelAPI[Movie]
+    person: DomainModelAPI[Person]
```

Comment:
Was this change required? Seems to work will in both cases.

Reply:
Just for testing purposes., I sort the order based on name to get consistent results.